### DISCUSSION TITLE
Scraper - Fetch current price from a substring?

### DISCUSSION THREAD

**Discussion by karter16** [ghostfolio/ghostfolio#2949] [Q&A] [answered]
Hi @dtslvr , I've been setting up the scraper which is all going fine, I have a question as to whether it is possible to retrieve a substring from the contents of an element.

In this example: https://www.morningstar.com.au/investments/security/fund/25426/quote the price is quoted as an "Entry/Exit" pair, and the buy and sell prices are in the same element like: "1.2345 / 1.2367". I've been looking at the Cheerio readme, and the Ghostfolio code and it looks to me as though Ghostfolio takes the contents of the element selected, performs a simple trim/strip out extra characters and then treat it as a number? I'm guessing in this case it's ending up with something like "1.23451.2367" which isn't valid? 

Am I correct in my assumption that at present there isn't a way to handle this scenario?

**Comment by dtslvr** [comment, marked as answer]
> Am I correct in my assumption that at present there isn't a way to handle this scenario?

Yes. The relevant code is in [ManualService](https://github.com/ghostfolio/ghostfolio/blob/main/apps/api/src/services/data-provider/manual/manual.service.ts#L257). Feel free to create a PR with a potential improvement.
